security/pkg/nodeagent/sds/server.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sds
+
+import (
+	"net"
+
+	"google.golang.org/grpc"
+
+	"istio.io/istio/pkg/log"
+)
+
+const maxStreams = 100000
+
+// Args provides all of the configuration parameters for secret discovery service.
+type Args struct {
+	SDSUdsSocket string
```

Comment:
Comment this field? It's a public option.

Reply:
Also could just be UDSPath? (UDS is already unix domain *socket*)

Reply:
thanks, updated.